### ISSUE TITLE
The password checkbox and the "1 download" button cannot be focused with "Tab" key

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla/send#754]
[Affected versions]:
- Firefox 58.0.1 and up

[Affected Platforms]:
- All Windows
- All Linux
- All Mac

[Steps to reproduce]:
1. Start Firefox browser and navigate to "https://send.stage.mozaws.net/" page.
2. Upload a file.
3. Press "Tab" key 3 times.
4. Observe the behavior.

[Expected result]:
- The checkbox is focused.

[Actual results]:
- The "Delete" button is focused.

[Notes]:
- This issue is also reproducible for the "1 download" button.
- Here is a screen recording with the issue:
![tab key - focus password checkbox](https://user-images.githubusercontent.com/31244235/35860556-312dde9e-0b4d-11e8-8ad7-0c6a54b92603.gif)

**Comment by dannycoates** [comment]
fixed as of 77e3b5a3e63c79f7d8d305f52da53a287b927bd4

**Comment by Softvision-CristinaBadescu** [comment]
This issue is no longer reproducible, the "1download" button and the require a password checkbox can be focused, using Firefox Send dev server, on latest Nightly build 61.0a1 (2018-03-28), on Windows 10 x64, Mac 10.13.3 and Arch Linux 4.12. 

However, if you press the "Enter" key when they are focused, nothing happens. I've filed the issue here: #797 .
